packages/@coorpacademy-components/src/molecule/progress-wrapper/style.css

```
@@ -127,6 +127,14 @@
   padding-top: 16px;
 }
 
+.toggleDetailsButton {
+  display: flex;
+  justify-content: flex-end;
+  cursor: pointer;
+  margin-top: 16px;
+  margin-bottom: 8px;
```

on privilégie le padding au margin
```suggestion
 padding: 16px 8px;
```